lib/Parser.js

```diff
@@ -1983,17 +1993,28 @@ class Parser extends Tapable {
 	}
 
 	getCommentOptions(range) {
+		const webpackCommentRegExp = new RegExp(
```

Comment:
Could you build the regex outside of the parser?

Reply:
@ooflorent agree. will do. thanks